Signal/src/ViewControllers/OWS2FARegistrationViewController.m

```diff
@@ -65,7 +74,6 @@ - (void)viewDidLoad
         @"REGISTER_2FA_INSTRUCTIONS", @"Instructions to enter the 'two-factor auth pin' in the 2FA registration view.");
 
     // Layout
-
     [entryView autoPinEdgesToSuperviewMargins];
```

Comment:
Since we're pinning to margins here, we're all good on iPhoneX

<img width="545" alt="screen shot 2018-03-05 at 11 47 32 am" src="https://user-images.githubusercontent.com/217057/36987788-3556c1a6-206b-11e8-96f0-6e332affb895.png">